docs/kbs_attestation_protocol.md

```diff
@@ -384,6 +386,50 @@ A request for protected resource can fail for three reasons:
 3. The requested resource does not exist. The KBS implementation sends an HTTP
    response with a 404 (`Not Found`) status code.
 
+### Using KBS-Provisioned Token
+
+After a successful Authentication process, a tee asymmetric keypair will be generated
+by the requester and KBS will provision a token that contains attestation results and
+endorsement for the tee public key.
+
+To request a protected resource from the KBS, the attester sends a `GET` request
+to a resource specific endpoint with the token as the bearer authorization. If the
+token passes verification, the KBS will respond to the `GET` request with an HTTP
+response which content is set to a [KBS Response](#response) JSON payload, whose
+encryption key is the one inside the token:
```

Comment:
Isn't the KBS response encrypted with a symetric key, that is wrapped with the attestation results TEE key?

Reply:
Yes, thanks. I added some more details about the declaration.